narratives/ncov_sit-rep_2020-05-01.md

```diff
@@ -0,0 +1,259 @@
+* [CDC Resources](https://www.cdc.gov/coronavirus/2019-ncov/index.html).
+* [Making sense of uncertainty](https://www.theatlantic.com/health/archive/2020/04/pandemic-confusing-uncertainty/610819/).
+<!-- There is no right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text -->
+# [A note on sampling](https://nextstrain.org/ncov/europe/2020-04-30?c=country&r=country&d=map&p=grid&legend=closed)
+We currently have sequences from samples collected across 6 continents. This is an incredible achievement -- sequencing an unknown, large RNA virus in the midst of a pandemic is difficult, and is only possible through the incredible work and timely sharing of data by scientists and physicians around the world.
+<br><br>
+While this data enables us to infer many useful characteristics of the outbreak and track its spread in real time, it's important to emphasize that our conclusions are limited by the available data.
+<br><br>
+For example, the map shows many samples from Iceland, the U.K., and the Netherlands. This does not necessarily indicate that SARS-CoV-2 is circulating more broadly in these areas; rather, we just have more data available from these countries. The size of each circle on the map indicates how much data is currently available from that area, rather than the true size of the outbreak.
+
+<!-- There is NO right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Many introductions to Russia](https://nextstrain.org/ncov/europe/2020-04-29?c=division&d=tree,map&f_country=Russia&p=full&r=division)
+
+Highlighting samples from Russia, we see that they are scattered both geographically (across Russia) and genomically (across the tree).
+<br><br>
+This indicates that the epidemic in Russia was sparked by many separate introductions, rather than a single introduction followed by within-country spread.
+<br><br>
+While there are a few samples towards the bottom of the tree that are most closely related to Asian cases, the majority of Russian samples descend from European lineages.
+
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Heterogeneous outbreak in Moscow](https://nextstrain.org/ncov/europe/2020-04-29?c=division&d=tree&f_country=Russia&f_division=Moscow&label=clade:A2&p=full&r=division&m=div)
+
+An example of frequent importation can be seen by zooming in on samples from Moscow.
+<br><br>
+There are a few clusters of cases which are most closely related to other samples from Moscow, indicating likely local transmission.
+However, most samples from Moscow are most closely related to samples from elsewhere, consistent with importation.
+<br><br>
+Given the number of confirmed cases in Moscow, it's likely that at least some of these introductions led to further local spread, but we don't yet have enough data to see this.
```

Comment:
How many cases are there in Moscow currently? Would be good to add in this in, like we did last week.